### ISSUE TITLE
Metamask version 8.0.6 does not send transactions to localhost dev chain(openethereum)

### ISSUE THREAD

**Issue by MeHighP** [MetaMask/metamask-extension#9179]
Google Chrome Version 84.0.4147.125 (Official Build) (64-bit)
Metamask version 8.0.6
opentethereum v3.0.0
Windows 10

Metamask won't send transactions to localhost 8545

**Comment by jacobcantele** [comment]
Hi @MeHighP, can you please share more detailed notes on this request? without more detailed notes on how to reproduce your issue, we won't be able to assist.

**Comment by tmashuang** [comment]
Possible related to https://github.com/openethereum/openethereum/issues/11349.

**Comment by MeHighP** [comment]
I just used Metamask UI to send some ETH to a particular address. Metamask is connected to localhost8545 which is in fact a local dev chain(openethereum).  After i click confirm transaction nothing happens. This worked with previous versions.  For example i have tested with 7.7.9 and it works just fine.
I have also found that the transactions are queued, see attachments(it seems that it tries port 8000 for some reason)
![CaptureMM1](https://user-images.githubusercontent.com/36454116/90392203-1da11180-e097-11ea-8ea8-c763ea615263.JPG)
![CaptureMM2](https://user-images.githubusercontent.com/36454116/90392219-242f8900-e097-11ea-954c-4e7e59dd6e2e.JPG)
